app/assets/stylesheets/helpers/_wrapper.scss

```diff
@@ -0,0 +1,5 @@
+@import "helpers/mixins";
+
+#wrapper {
```

Comment:
👍  Indeed, wrapper styles should only be used/needed by the wrapper_layout (which we hope to remove rather soon from all remaining applications – be that frontend and licence-finder).